FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Models/Jobs/Job.cs

```diff
@@ -44,9 +47,34 @@ public class Job
             {
                 _ = ResourceProgresses.TryAdd(resource, null);
                 _ = TotalResourceCounts.TryAdd(resource, 0);
-                _ = ProcessedResourceCounts.TryAdd(resource, 0);
-                _ = SkippedResourceCounts.TryAdd(resource, 0);
-                _ = PartIds.TryAdd(resource, 0);
+
+                // Temporarily set schema type list only contains single value for each resource type
+                // E.g:
+                // Schema list for "Patient" resource is ["Patient"]
+                // Schema list for "Observation" resource is ["Observation"]
+                _ = SchemaTypeMap.TryAdd(resource, new List<string>() { resource });
```

Comment:
I think here is a kind of config, users can set up the target schema types for each resources, even if we don't support that in ARM template. When pipeline running it will try to get the schema content according to schemaManager

Reply:
Agree that is a kind of config. What about putting this schemaTypeMap in schemamanager project?

Reply:
Moved **schemaTypesMap** to **schemaManager**